docs/sites.yml

```diff
@@ -8334,3 +8334,14 @@
   built_by: Peter Brady
   built_by_url: https://www.peterbrady.co.uk
   featured: false
+- title: Salexa - Estetica Venezolana
+  url: https://peluqueriavenezolana.cl/
+  main_url: https://peluqueriavenezolana.cl/
+  source_url: https://github.com/enbonnet/salexa-front
+  description: >
+    Venezuelan beauty or hairdressing site in Chile
```

Comment:
Should it be
```suggestion
    Venezuelan beauty and hairdressing salon in Chile
```
?

Reply:
Yes, the suggestion is good, my confusion was to talk about the place and not the website, now it's better